src/perl/dmtest/DMTest/Separated.pm

```diff
@@ -48,5 +55,4 @@ sub suite {
   }
   return $suite;
 }
-
```

Comment:
Shoot, forgot to mention this: could you restore the blank line here, before the final 1; please?

Reply:
fixed